feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go

```diff
@@ -139,11 +139,16 @@ func verifyNodeTelemetry(t *testing.T, node, peer gnmi.DeviceOrOpts, nodePort, p
 	if !lldpEnabled {
 		if _, ok := gnmi.Watch(t, node, interfacePath.State(), time.Minute, func(val *ygnmi.Value[*oc.Lldp_Interface]) bool {
 			intf, present := val.Val()
-			if !present {
-				return false
+
+			if intf == nil {
+				return true
+			} else {
+				if !present {
```

Comment:
The predic func expects `true` when the interface is not present, right? 

I think https://github.com/openconfig/featureprofiles/pull/954 is good enough?

Reply:
Agree. I will close this pull request. Thanks.